src/parkapi_sources/converters/goldbeck/converter.py

```diff
@@ -0,0 +1,60 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+from datetime import datetime, timezone
+from typing import Any
+
+from openpyxl.cell import Cell
+
+from parkapi_sources.converters.base_converter.push import NormalizedXlsxConverter
+from parkapi_sources.models import SourceInfo
+
+
+class GoldbeckPushConverter(NormalizedXlsxConverter):
+    source_info = SourceInfo(
+        uid='goldbeck',
+        name='GOLDBECK Parking Services GmbH',
+        public_url='https://www.goldbeck-parking.de',
+        has_realtime_data=False,
+    )
+
+    purpose_mapping: dict[str, str] = {
+        'Auto': 'CAR',
+        'Fahrrad': 'BIKE',
+    }
+
+    supervision_type_mapping: dict[str, str] = {
+        True: 'YES',
+        False: 'NO',
+    }
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # For some reason, Goldbeck decided to change the titles
+        goldbeck_header_rows: dict[str, str] = {
+            'Einfahrtshöhe (cm)': 'max_height',
+            'Zweck der Anlage': 'purpose',
+            'Überdachung': 'is_covered',
+            'Art der Überwachung': 'supervision_type',
+        }
+        self.header_row = {
+            **{key: value for key, value in super().header_row.items() if value not in goldbeck_header_rows.values()},
+            **goldbeck_header_rows,
+        }
+
+    def map_row_to_parking_site_dict(self, mapping: dict[str, int], row: list[Cell]) -> dict[str, Any]:
```

Comment:
Why did you copy paste instead of just using `result = super().map_row_to_parking_site_dict(...)`, and then do custom modifications afterwards?

Reply:
Could you please elaborate the custome modifications or do you have an example on how to go about the custom modifications? ```super().map_row_to_parking_site_dict(mapping: dict[str, int], row: list[Cell])``` requires 2 parameters, which are ```mapping``` and ```row```. How do I reference them in the converter.py?

Reply:
`result = super().map_row_to_parking_site_dict(mapping, row)`, and afterwards doing the custom modifications at `result`.

Reply:
... just tried it myself and saw the issue in detail. With your code, you just remove all opening times. Some of then "Klinik am Gesundbrunnen" is so broken that we cannot repair it anyway. So is it by purpose that you removed all opening times?

Reply:
Decision: merge without opening times, add opening times later: https://github.com/ParkenDD/parkapi-sources-v3/issues/77